core/bones/password.py

```diff
@@ -90,13 +90,13 @@ def __init__(
                     smaller this will be invalid.
 
             :param test_threshold: The minimum number of tests the password must pass.
-            :param password_tests: A list of tuple.The tuple  contains the tests and an error message if the test fails
+            :param password_tests: A list of tuples. The tuple contains the test and a reason for the user if the test
+                    fails.
         """
         super().__init__()
         self.min_password_length = min_password_length
         self.test_threshold = test_threshold
-        if if password_tests is not None:
-            self.password_tests = password_tests
+        self.password_tests = password_tests
```

Comment:
@ArneGudermann Sorry for the `if if` in my suggestion. But now you would apply also `password_tests` if they are `None`.